taskcluster/scripts/fetch/enter_dev_shell.ps1

```diff
@@ -8,7 +8,7 @@
 
 $VS_STUDIO_LOCATION =resolve-path $PSScriptRoot
 $W10_SDK_PATH =  resolve-path "$PSScriptRoot\SDK"
-$W10_SDK_VERSION = "10.0.19041.0"
+$W10_SDK_VERSION = "10.0.20348.0"
```

Comment:
Why does the SDK version need to change if 
```if(NOT MSVC)```
msvc is not touched?

Reply:
The version we use of win 10 SDK generates c++ warnings. We should use this version to enable WError on windows.
I would like to know the process for updating the SDK.

Reply:
I see :) - 
This script generates the bundle, however given it requires root we can't run it in CI. 
Therefore we get the bundle in a fetch task. The next version we generate should live on tooltool - preferably :) 
https://github.com/mozilla-mobile/mozilla-vpn-client/blob/51876bf0021a9df290d984e48b16cd4fcc36143a/taskcluster/ci/fetch/kind.yml#L16-L25